test/server/core/Game.spec.js

```diff
@@ -0,0 +1,45 @@
+describe('Overall game mechanics', function() {
+    integration(function() {
+        describe('Simultaneous lethal damage to both bases', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        leader: 'sabine-wren#galvanized-revolutionary',
+                        base: { card: 'chopper-base', damage: 29 }
+                    },
+                    player2: {
+                        base: { card: 'administrators-tower', damage: 29 }
+                    }
+                });
+            });
+
+            it('should result in the game ending in a draw', function () {
+                this.player1.clickCard(this.sabineWren);
+                this.player1.clickPrompt('Deal 1 damage to each base');
+                expect(this.player1).toHavePrompt('Tie Game');
+                expect(this.player2).toHavePrompt('Tie Game');
+            });
+        });
+
+        describe('One player\'s base taking lethal damage', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['rebel-pathfinder']
+                    },
+                    player2: {
+                        base: { card: 'administrators-tower', damage: 29 }
+                    }
+                });
+            });
+
+            it('should cause that player to lose the game', function () {
+                this.player1.clickCard(this.rebelPathfinder);
+                expect(this.player1).toHavePrompt('Game Won');
```

Comment:
Can we find a way to check which player won, and do that check for both players' prompts?

Reply:
Right now the winner is printed in the prompt's menuTitle. I'm not really sure what menuTitle and promptTitle will correspond to in the actual UI, so: should I also add the winner's name to the promptTitle or should I check the menuTitle in the test somehow?

Reply:
I think `toHavePrompt` will pass if it matches either promptTitle or menuTitle